dipy/reconst/shm.py

```diff
@@ -965,3 +965,80 @@ def sh_to_sf_matrix(sphere, sh_order, basis_type=None, return_inv=True, smooth=0
         return B.T, invB.T
 
     return B.T
+
+def sh_to_ap(coeffs_mtx, normal_factor=0.00001):
+    """ Calculates anisotropic power map with a given SH coefficient matrix
+
```

Comment:
I would write sh_to_anisotropic_power if ap is misleading. Or just say 
```
def anisotropic_power(
```
without the sh.

And yes as @mdesco suggests the coeffs_mtrx name has to change.